pkg/tfgen/docs.go

```diff
@@ -598,9 +598,9 @@ func (p *tfMarkdownParser) parseSection(section []string) error {
 		return nil
 	case "Example Usage":
 		sectionKind = sectionExampleUsage
-	case "Arguments Reference", "Argument Reference", "Nested Blocks", "Nested blocks":
+	case "Arguments Reference", "Argument Reference", "Argument reference", "Nested Blocks", "Nested blocks":
```

Comment:
Maybe header should just be lower/uppercased before being switched on?

Reply:
This is ALL being replaced! This is just a quick fix for now to unblock an issue with a provider